### PR TITLE
Add tests for fmt and keys

### PR DESCRIPTION
**Pull request by ZichengMa**

### CODE REVIEW

**Comment by marshtompsxd** [comment]
@ZichengMa Does this cover ALL external/external_body functions in kubernetes_api_objects?

File: src/unit_tests/kubernetes_api_objects/toleration copy.rs
Comment (marshtompsxd) [inline src/unit_tests/kubernetes_api_objects/toleration copy.rs:1]: What is this file for? The name is strange

Comment (ZichengMa) [reply src/unit_tests/kubernetes_api_objects/toleration copy.rs:1]: Mistakenly copied it. I will delete it.

**Comment by ZichengMa** [comment]
> @ZichengMa Does this cover ALL external/external_body functions in kubernetes_api_objects?
I think so. I will update a python script later to check.

File: src/unit_tests/kubernetes_api_objects/toleration copy.rs
Comment (marshtompsxd) [reply src/unit_tests/kubernetes_api_objects/toleration copy.rs:1]: Can you delete it so I can merge the PR?

**Pull request merged**